Ports/aria2/package.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='aria2'
+version='1.36.0'
+files="https://github.com/aria2/aria2/releases/download/release-${version}/aria2-${version}.tar.xz ${port}-${version}.tar.xz 58d1e7608c12404f0229a3d9a4953d0d00c18040504498b483305bcb3de907a5"
+auth_type="sha256"
+depends=("libssh2" "libxml2" "openssl" "zlib" "libuv")
```

Comment:
Please break this into one per line

Reply:
Are you sure? I can't see any other package.sh scripts that do that.

Reply:
Yes, it creates more meaningful diffs/git blame results :)